documentation/index-for-docs.html

```diff
@@ -0,0 +1,17 @@
+<html>
```

Comment:
This is a fairly ugly index.html I added just in case someone happens to land at the top level of the docs. But I think we should add direct links to each doc from our platform docs, so it will be highly unlikely anyone will land here.